concourse/scripts/ssh.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+
+key=/tmp/concourse_id_rsa.$RANDOM
+trap "rm -f $key" SIGINT SIGTERM EXIT
+
+eval $(make dev showenv | grep CONCOURSE_IP=)
+
+aws s3 cp "s3://${DEPLOY_ENV}-state/concourse_id_rsa" $key && chmod 400 $key
+
+echo
+aws s3 cp "s3://${DEPLOY_ENV}-state/generated-concourse-secrets.yml" - | \
+ruby -ryaml -e 'puts "\Sudo password is " + YAML.load(STDIN)["secrets"]["concourse_vcap_password_orig"]' 
```

Comment:
To have one less dependency. I wanted to make this as simple as possible & clear what it is doing.